tests/unit/test_genotype_phenotype.py

```diff
@@ -7,16 +7,19 @@
 from __future__ import unicode_literals
 
 import unittest
+import rdflib
 
 import ga4gh.datamodel.genotype_phenotype as g2p
 import ga4gh.datamodel.datasets as datasets
 
+# from ga4gh.protocol import FeaturePhenotypeAssociation
+
 
 class TestPhenotypeAssociationSet(unittest.TestCase):
     def setUp(self):
         ds = datasets.AbstractDataset("test")
         self.phenotypeAssocationSet = g2p.PhenotypeAssociationSet(
-            ds, "test", None)
+            ds, "test", '/server/tests/data/datasets/dataset1/phenotypes/cgd')
```

Comment:
I'll test that option. In case it fails, instead of hardcoding to docker's
path, I'll autodetect the path based on current directory and figure out
the path from there.

Personal blog <http://blog.gabrielsaldana.org>
Notasbit - Noticias de tecnologia en español <http://www.notasbit.com>

On Thu, Mar 17, 2016 at 12:39 PM, Brian <notifications@github.com> wrote:

> In tests/unit/test_genotype_phenotype.py
> <https://github.com/ohsu-computational-biology/server/pull/26#discussion_r56556609>
> :
>
> >
> >  class TestPhenotypeAssociationSet(unittest.TestCase):
> >      def setUp(self):
> >          ds = datasets.AbstractDataset("test")
> >          self.phenotypeAssocationSet = g2p.PhenotypeAssociationSet(
> > -            ds, "test", None)
> > +            ds, "test", '/server/tests/data/datasets/dataset1/phenotypes/cgd')
>
> This is the correct path? It works.... At the same time I would have
> guessed the path would start with "tests/data/..."
> I'm thinking that this works because /server is true for our docker
> image. I did a quick experiment. It works with the path set to "tests/..."
> can you try it on your side and if it works, change it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ohsu-computational-biology/server/pull/26/files/8d309f5ee65e30f483cb977dc6707b2cdfdbae35..18443092a8fa2c02667bc9e993d7e56e5e4eb731#r56556609>
>